prime-router/src/main/kotlin/history/db/DeliveryHistoryDatabaseAccess.kt

```diff
@@ -266,11 +264,37 @@ class DeliveryHistoryDatabaseAccess(
             .where(whereClause).asTable(DeliveryHistoryTable.DELIVERY_HISTORY)
 
         val results = db.transactReturning { txn ->
-            search.fetchResults(
+
+            val results = search.fetchResults(
```

Comment:
I guess Kotlin was okay with it but having another results declared here feels weird with the variable two lines up and doing something like `results.results` a few lines below. Maybe historyResults or something like that?

Reply:
Good catch!  I wonder why intellij isn't giving me that shadowed varname warning 🤷🏻 .  Changed to `historyResults`